Test/dafny0/UserSpecifiedTypeParameters.dfy.expect

```diff
@@ -1,6 +1,6 @@
 UserSpecifiedTypeParameters.dfy(26,10): Error: the type of this variable is underspecified
-UserSpecifiedTypeParameters.dfy(26,24): Error: type of type parameter could not be determined; please specify the type explicitly
-UserSpecifiedTypeParameters.dfy(26,24): Error: type of type parameter could not be determined; please specify the type explicitly
+UserSpecifiedTypeParameters.dfy(26,16): Error: type of type parameter could not be determined; please specify the type explicitly
```

Comment:
Here the position 24 was more appropriate, especially for a quick fix. Could you please revert that?

Reply:
24 points to `H`, while 16 points to `MyClass`. The error is about missing type argument for `MyClass`, so why do you prefer the previous message?

Reply:
Oh you're right. I forgot that characters positions start at 0, so I thought I was pointing to the dot, which is exactly the place where to insert type parameters. In this case, this fix is better.